style/app.scss

```diff
@@ -0,0 +1,2407 @@
+      }
+      > p {
+        // font-size: 16px;
+        // font-weight: 400;
+        text-align: center;
+        max-width: 31.4375rem;
+        color: $text-secondary;
+        margin: 0.75rem auto 2.5rem;
+        @media screen and (max-width: 1024px) {
+          font-size: 0.75rem;
+          max-width: 24rem;
+          margin: 0.5rem auto 1.5rem;
+        }
+        @media screen and (max-width: 768px) {
+          max-width: 20.875rem;
+        }
+        @media screen and (max-width: 575px) {
+          max-width: 24.0625rem;
+        }
+        @media screen and (max-width: 480px) {
+          max-width: 13.6875rem;
+        }
+      }
+      > span {
+        display: block;
+        font-size: 1.125rem;
+        font-weight: 400;
+        width: 11.25rem;
+        text-align: center;
+        border-bottom: 0.5px solid $color-primary;
+        margin: auto;
+        padding-bottom: 0.75rem;
+        @media screen and (max-width: 1024px) {
+          font-size: 0.75rem;
+          width: 8.625rem;
+          padding-bottom: 0.5rem;
+        }
+        @media screen and (max-width: 768px) {
+          width: 6.25rem;
+          padding-bottom: 0.375rem;
+        }
+        @media screen and (max-width: 575px) {
+          font-size: 0.625rem;
+          padding-bottom: 0.25rem;
+        }
+        @media screen and (max-width: 480px) {
+          font-size: 0.75rem;
+          width: 8.375rem;
+          padding-bottom: 0.5rem;
+        }
+      }
+      .filtered-card-listing {
+        display: grid;
+        grid-template-columns:  auto auto auto;
+        gap: 20px;
+        padding: 1.5rem 1.5625rem 0;
+        @media screen and (max-width: 1199px) {
+          gap: 16px;
+        }
+        @media screen and (max-width: 1024px) {
+          padding: 1.0625rem 0 0;
+        }
+        @media screen and (max-width: 575px) {
+          grid-template-columns: auto auto;
+        }
+        @media screen and (max-width: 480px) {
+          grid-template-columns: auto;
+          padding: 1rem 0;
+        }
+        .card {
+          position: relative;
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+          border-radius: 12px;
+          box-shadow: 0 0 15px 0 $shadow-color;
+          padding: 24pzx 1.5rem 22zpx;
+          @media screen and (max-width: 1024px) {
+            border-radius: 8px;
+            padding: 0.8125rem 0.75rem 0.75rem 0.6875rem;
+          }
+          @media screen and (max-width: 768px) {
+            padding: 0.625rem 0.375rem 0.5625rem 0.4375rem;
+          }
+          @media screen and (max-width: 575px) {
+            padding: 1.25rem 1rem 0.75rem 0.75rem;
+          }
+          @media screen and (max-width: 480px) {
+            padding: 1rem 0.8125rem 0.6875rem 0.875rem;
+          }
+          .rank {
+            position: absolute;
+            font-size: 1rem;
+            font-weight: 600;
+            top: 0;
+            left: 1.5rem;
```

Comment:
don't use rem units for position properties like top,left, right,bottom

Reply:
i have checked the complete code and chnaged